src/Merger/SchemaMergerInterface.php

```diff
@@ -39,10 +39,10 @@ public function merge(): int;
     public function exportSchema(SchemaTemplateInterface $rootRootSchemaTemplate): void;
 
     /**
-     * @param  array<string,mixed> $schemaDefinition
-     * @return array<string,mixed>
+     * @param  string|array<string,mixed> $schemaDefinition
+     * @return string|array<string,mixed>
      */
-    public function transformExportSchemaDefinition(array $schemaDefinition): array;
+    public function transformExportSchemaDefinition($schemaDefinition);
```

Comment:
i will check if we can somehow avoid this, this would already mean another mayor :smile: